packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -37,19 +38,25 @@ contract SuperchainWETH is WETH98, ICrosschainERC20, ISemver {
         super.withdraw(_amount);
     }
 
+    /// @inheritdoc WETH98
+    function allowance(address src, address guy) public view override returns (uint256) {
```

Comment:
should we replace `src` for `owner` and `guy` for `spender` as in `IWETH`? 

it feels more consistent in this contract